### PR TITLE
Add tctl next deprecation message

### PR DESCRIPTION
**Pull request by feedmeapples**
## What was changed

Added a `tctl next` deprecation warning

## Why?

tctl next has been moved to https://github.com/temporalio/cli

## Checklist

1. Closes 

2. How was this tested:

image

3. Any docs updates needed?

### CODE REVIEW

**Review by bergundy — COMMENTED** [review]
We should deprecate tctl entirely, there’s no point in maintaining 2 clis

**Comment by bergundy** [comment]
It might be a bit too soon to deprecate this entirely, let’s come up with a checklist in the new year and do it

**Comment by feedmeapples** [comment]
> It might be a bit too soon to deprecate this entirely, let’s come up with a checklist in the new year and do it

should we deprecate + remove tctl-next first and then add message for tctl v1 deprecation? I want to remove the code of tctl-next from here and remove the dependency on tctl-kit which has breaking changes

**Comment by robholland** [comment]
Maybe replace `tctl` next behaviour with printing a message saying to install Temporal CLI, with relevant `brew` and `install.sh` suggestions.

File: cli/app.go [hunk 1]
```diff
@@ -140,6 +139,17 @@ func NewCliApp() *cli.App {
 	app.After = stopPlugins
 	app.ExitErrHandler = handleError
 
+	tctlNextDeprecation := fcolor.YellowString("WARNING: tctl next is going to be deprecated. Consider switching back to tctl v1 or upgrading to Temporal single binary https://github.com/temporalio/cli#getting-started\n")
```
Comment (bergundy) [inline cli/app.go:142]:
```suggestion
	tctlNextDeprecation := fcolor.YellowString("WARNING: tctl next is going to be deprecated. Please upgrade to Temporal CLI https://github.com/temporalio/cli#getting-started\n")
```

**Review by bergundy — APPROVED** [review]

**Pull request merged**